lib/active_fedora/model.rb

```diff
@@ -5,6 +5,45 @@ module ActiveFedora
   # This module mixes various methods into the including class,
   # much in the way ActiveRecord does.
   module Model
+    def self.best_class_from_uris(uris, default: nil)
```

Comment:
I'm wondering if this can be tightened up a bit by removing the primitive as a default, like the following (but I'm not sure a case or two don't fall out of this one):

```ruby
def self.best_class_from_uris(uris, default: ActiveFedora::Base)
  best_model_match = default

  uris.each do |uri|
    model_value = from_class_uri(uri)
    next unless model_value

    # If there is an inheritance structure, use the most specific case.
    best_model_match = model_value if best_model_match > model_value
  end

  best_model_match
end
```